projectile.el

```diff
@@ -736,6 +741,10 @@ A thin wrapper around `file-truename' that handles nil."
       (projectile-project-root)
     (error nil)))
 
+(defun projectile-default-project-name (project-root)
+  "Default function used create project name to be displayed"
```

Comment:
`M-x checkdoc` won't like this. `project-root` must appear in all-uppercase in the docstring and it should end with a `.`.